### ISSUE TITLE
Plugin Constantly Hangs While Trying To Make Changes to Profiles.

### ISSUE THREAD

**Issue by CaptainAS007** [nguyenquyhy/Flight-Tracker-StreamDeck#383]
I had originally installed the version from the Marketplace. I uninstalled it and and did an manual install to a newer version from github and started having this problem.

I cannot tell what version is really loaded. /@appdata@/Local/Elgato/Streamdeck doesn't have a Plugin folder, but I see com.elgato.obs and others not in a plugin folder. But not flight Tracker. I also looked in Programs and can't find it there.

A great enhancement would be to be able what version is being used from Stream Deck App.

Since I can't find that plugin, I can't send any log file. 

Where can it be? Any Idea where else I can look to find and manually delete so I can reinstall.

BTW, you've done a great job with this plugin, I really enjoy using it!!

**Comment by nguyenquyhy** [comment]
> I cannot tell what version is really loaded

Just need to right click any button and you'll see the version.
![image](https://user-images.githubusercontent.com/1560418/212356359-a50f9c2f-963a-46e0-9b09-523c2aef24f4.png)


> /@appdata@/Local/Elgato/Streamdeck

Are you sure you tried `%appdata%`, not /@appdata@. The former is a special shortcut, while the latter shouldn't exist at all.
Here is the full path `%appdata%\Elgato\StreamDeck\Plugins`.

**Comment by CaptainAS007** [comment]
I did right click and it shows .45 which is the one from the Marketplace which I did an uninstall from the Marketplace and tried to manually installed .48.

I meant to put % instated of %appdata%, should be under users right? That one has no Plugin, only the Program files.

I tried to uninstall both from right clicking and from the marketplace and it actually will not uninstall. I didn't notice that when I did the uninstall and then juts when and installed manually. But I guess, that didn't override the old .45 version and I can't find it to manually remove it.

Thanks for your quick response!!!!

**Comment by nguyenquyhy** [comment]
You are looking at the wrong places. Just put the full path I gave into the address bar or Run (Windows + R) and you'll see the correct place.
`%appdata%\Elgato\StreamDeck\Plugins`

**Comment by CaptainAS007** [comment]
Ahh. I assumed that it was in local is in Roaming. I thought I had looked in there too, but I guess I didn't.

![image](https://user-images.githubusercontent.com/121839906/212440679-90b10836-c4f7-46ed-904a-1fcbe85e7a69.png)
![image](https://user-images.githubusercontent.com/121839906/212440737-b92c0283-66b2-4457-bc8a-d42966df04fd.png)

**Comment by nguyenquyhy** [comment]
So is it still hanging? Have you found the log files?

**Comment by CaptainAS007** [comment]
Sorry for the long delay responding, hadn't been flying. Yes, it still hangs, but in doing google searches I found the problem.

Some people were saying that it couldn't be connected to a hub or USB 3 port. That a direct connect to USB 2 was required. I did that and still was hanging. 

I then found that another guy mention he was also having issues and it always happened when the Stream Deck application was open in a monitor other that your primary monitor. I have three monitors. I started opening it only on the main monitor and the problem is indeed related to it not working if not used in your main monitor. I can easily recreate the problem when I move the app to one of my other monitors.

I will report the problems to stream deck.

But, I still love you plugin and it works great! I really appreciate your help in getting me to the latest version!
